docs/src/main/asciidoc/spring-cloud-gcp.adoc

```diff
@@ -14,13 +14,13 @@ The Spring Cloud GCP project aims at making the Spring Framework a first-class c
 
 Currently, Spring Cloud GCP lets you leverage the power and simplicity of the Spring framework to:
 
-1. Publish and subscribe from Google Cloud Pub/Sub topics
-2. Configure Spring JDBC with a few properties to use Google Cloud SQL
-3. Write and read from Spring Resources backed up by Google Cloud Storage
-4. Exchange messages with Spring Integration using Google Cloud Pub/Sub on the background
-5. Trace the execution of your app with Spring Cloud Sleuth and Google Stackdriver Trace
-6. Configure your app with Spring Cloud Config, backed up by the Google Runtime Configuration API
-7. Consume and produce Google Cloud Storage data via Spring Integration GCS Channel Adapters
+. Publish and subscribe to Google Cloud Pub/Sub topics
```

Comment:
Since we are here, would you mind also fixing a `Spring Cloud GCP Reference Documentation` to something like just `Spring Cloud GCP`?
The `Reference Documentation` doesn't play good in the aggregated doc of Spring Cloud: https://cloud.spring.io/spring-cloud-static/Greenwich.RC1/multi/multi_spring-cloud.html

Pay attention to other projects links. They don't have such a noise in their names.

Thanks

Reply:
Sure. Done!